### DISCUSSION TITLE
[@types/vimeo__player] missing new embed options

### DISCUSSION THREAD

**Discussion by petr-hajek** [DefinitelyTyped/DefinitelyTyped#70333] [Issues with a @types package]
New options were added to Vimeo player: https://github.com/vimeo/player.js/releases
https://developer.vimeo.com/player/sdk/embed
Please update the types so the Player.Options type is up to date

[a comment by an automated account is not shown]

**Comment by oom-** [comment]
Missing: `mozallowfullscreen` props: 
 
https://developer.vimeo.com/api/oembed/videos#embedding-a-video-with-oembed-step-2
- `webkitallowfullscreen`  ❌
- `mozallowfullscreen` ❌ 
- `allowfullscreen`  ❌
